### PR TITLE
Feature: Show or Hide the Terminal Display

### PR DESCRIPTION
**Pull request by noroadsleft**
Adds a toggle control that collapses (and re-expands!) the terminal display.

[Here's a sample video.](https://cdn.discordapp.com/attachments/456176332825427969/645222311330578444/2019-11-16_03-13-13.mp4)

### CODE REVIEW

File: src/components/StatusPanel.vue [hunk 1]
```diff
@@ -22,6 +27,21 @@ export default {
...
       });
+    },
+    /**
+     * toggleTerminal keymap. Attempts to load the keymap data from
+     * a predefined known file path.
+     * @param {boolean} isAutoInit If the method is called by the code
+     * @return {object} promise when it has completed
+     */
+    toggleTerminal() {
+      if (document.getElementById('terminal').getAttribute('class')) {
```
Comment (zekth) [inline src/components/StatusPanel.vue]: could you do it the vue way? like: https://vuejs.org/v2/guide/class-and-style.html

File: src/components/StatusPanel.vue [hunk 2]
```diff
@@ -32,3 +52,49 @@ export default {
   }
 };
 </script>
+<style>
+#status {
```
Comment (zekth) [inline src/components/StatusPanel.vue]: Please move this in style.scss to be compiled during the build process

Comment (noroadsleft) [reply src/components/StatusPanel.vue]: Just the `#status` rule, or the whole style sheet?

Comment (zekth) [reply src/components/StatusPanel.vue]: the whole sheet

Comment (noroadsleft) [reply src/components/StatusPanel.vue]: 53b8d6bc36c692d5896a62dcdc87a0c6e11ed93d

File: src/components/StatusPanel.vue (hunk 1 above)
Comment (noroadsleft) [reply src/components/StatusPanel.vue]: 2aba62ef78e42e1e87323fa0fadc08f8bde78b1a?

**Comment by zekth** [comment]
A last formating command and it'll be ok :)

**Comment by noroadsleft** [comment]
@zekth Yeah, making some polishing tweaks and then will cap off with a linting.

At some point I should track down why Prettier gives me different results than I get from Travis CI.

**Pull request merged**